README.md

```diff
@@ -1,3 +1,14 @@
 Testing example for begginers
 
+S-Single responsibility principle: This concept say that every class only need must 	will have one job, that’s mean that the system should be separate in 		different modules and everyone should will have a job. That help to make 		easier at the moment when in necessary change the code, because the module 		that need the change is the only that will change.
+
+O-Open/closed principle: This principle mean the classes, modules or functions should 	be “open” by an	 “extension” to allow add code or elements at the classes.
+	Also say that the classes should be closed when other classes have access to 	them, that’s mean that the classes that will called must be are correct and 	don’t have errors. The inheritance it’s based in this principle.
+
```

Comment:
classes that will call must be correct. You should not use words in past when you use 'will' unless you use 'have word too.